src/components/pdf-viewer/PDFViewer.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Spinner from 'reactstrap/es/Spinner'
+
+import 'components/pdf-viewer/PDFViewer.scss'
```

Comment:
It's better to import this kind of stuff locally. It makes you understand how styles are resolved, allows to move the component to different place without pain and prevents accidental forgetting about style file.

Reply:
Just scss files or everything should be imported locally?

Reply:
In this case, CSS, because it's coupled to to the component code.

In general, the guideline is following:
- if the component is widely used, is a part of library or can be decoupled from the project, it's easier to have an alias
- if the component cannot be decoupled, it's better to have clear path in the editor to understand where the file are

Reply:
yes, it makes sense. Feel free to point out wrong imports since I am not used to this guideline. To be honest I would rather prefer to use only local   or absolute imports not a mix of it.